lib/snapshot/snapshot.go

```diff
@@ -12,10 +13,18 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/utils"
```

Comment:
Imports from `lib/..` packages should not import packages from `app/...` packages as the idea is to do this another way around. All re-usable code should be a part of `lib`.
Could you please move `app/vmalert/utils/tls.go` to `lib/httputils` and update references to it here and in `app/vmalert` to point to a new location?